docs/api.rst

```diff
@@ -9340,6 +9340,118 @@ Editing blockchain account data
    :statuscode 409: An account given to edit does not exist or a given tag does not exist.
    :statuscode 500: Internal rotki error
 
+
+Account operations by chain type
+==================================
+
+.. http:patch:: /api/(version)/blockchains/type/(chain_type)/accounts
+
+   .. note::
+      Supported blockchains types: ``EVM, BITCOIN, SUBSTRATE``
+
+   .. note::
+      This endpoint doesn't support ENS resolution
+
+   Doing a PATCH on this endpoint with a list of accounts to edit will edit the label and tags for those accounts in all the chains of the same type where they are tracked.
+ 
+   **Example Request**:
+
+   .. http:example:: curl wget httpie python-requests
+
+      PATCH /api/1/blockchains/evm/accounts HTTP/1.1
+      Host: localhost:5042
+      Content-Type: application/json;charset=UTF-8
+
+      {
+          "accounts": [
+              {
+                  "address": "0x78b0AD50E768D2376C6BA7de33F426ecE4e03e0B",
+                  "label": "my new metamask",
+                  "tags": [
+                      "public",
+                      "metamask"
+                  ]
+              },
+              {
+                  "address": "0xc37b40ABdB939635068d3c5f13E7faF686F03B65",
+                  "label": "my hardware wallet"
+              }
+          ]
+      }
+
+
+   :reqjson list[object] accounts: A list of account data to edit.
```

Comment:
Shouldn't you mention the blockchain type here too?

Reply:
We don't do it for other endpoints since it is a variable in the URL and I've documented in the note the valid types